codec-http2/src/main/java/io/netty/handler/codec/http2/HpackHuffmanDecoder.java

```diff
@@ -230,15 +230,14 @@ AsciiString end() throws Http2Exception {
         }
 
         private void append(int i) {
-            try {
-                bytes[index] = (byte) i;
-            } catch (IndexOutOfBoundsException ignore) {
+            if (bytes.length >= index) {
```

Comment:
is `index == bytes.length` enough? (wondering if we can be consistent with https://github.com/netty/netty/pull/6845).